pkgs/objective_c/macos/Classes/objective_c.c

```diff
@@ -3,6 +3,6 @@
 // BSD-style license that can be found in the LICENSE file.
 
 // Relative import to be able to reuse the C sources.
-// See the comment in ../{projectName}}.podspec for more information.
```

Comment:
@dcharkes Looks like a bug in the plugin_ffi template?

Reply:
Sorry, I'm missing context here. Is the comment outdated? Or is something not working?

(The plugin_ffi template should be tested on Flutter CI, so it must be working.)

Reply:
This comment line was generated by the plugin_ffi template:

```c
// See the comment in ../{projectName}}.podspec for more information.
```

The `projectName` substitution didn't work here. I think there's a missing `{` at the start.

Reply:
https://github.com/flutter/flutter/pull/148378